go/multipooler/manager/manager.go

```diff
@@ -159,6 +160,27 @@ func (pm *MultiPoolerManager) unlock() {
 	pm.actionSema.Release(1)
 }
 
+// InitializeConsensusState initializes the consensus state if not already initialized.
+// This should be called when the consensus service is enabled.
+func (pm *MultiPoolerManager) InitializeConsensusState() {
+	pm.mu.Lock()
+	defer pm.mu.Unlock()
+
+	if pm.consensusState == nil {
+		pm.consensusState = NewConsensusState(pm.config.PoolerDir, pm.serviceID)
+
+		// Load existing consensus term from disk if it exists
+		if err := pm.consensusState.Load(); err != nil {
+			pm.logger.Error("Failed to load consensus state from disk", "error", err)
+			// Don't return error here - we'll initialize with default values (term=0)
```

Comment:
Seems like there are two reasons for failing to read the file:
- A newly created shard has no consensus history. In this case we want to create a new file
- Somehow access to the consensus file was lost or corrupted (disk issue, permissions issue, operator error, etc.). In this case it seems less obvious how to safely recover. If we acknowledged joining a term and that state is corrupted on disk, maybe the safest option is fail pooler startup, similar to what we would do if postgres data on disk was unreadable?...